.github/CODEOWNERS

```diff
@@ -2,11 +2,20 @@
 # last matching pattern is used, see
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax
 
-* @oss-review-toolkit/kotlin-devs
-*.md @oss-review-toolkit/core-devs
-/**/*Sw360* @oss-review-toolkit/kotlin-devs @bs-ondem @oheger-bosch
-/advisor/ @oss-review-toolkit/kotlin-devs @MarcelBochtler @oheger-bosch
-/clients/fossid-webapp/ @oss-review-toolkit/kotlin-devs @nnobelis
+# Members of @oss-review-toolkit/kotlin-devs.
+* @fviernau @MarcelBochtler @mnonnenmacher @nnobelis @oheger-bosch @sschuberth
+
+# Members of @oss-review-toolkit/core-devs.
```

Comment:
Also here, I'd prefer to more general comment to describe the idea, like "Contributors who usually review documentation."